doc/saw-user-manual/analyzing-hardware-circuits-using-yosys.md

```diff
@@ -123,19 +134,19 @@ sawscript> enable_experimental
 sawscript> m <- yosys_import "adder.json"
 sawscript> :type m
 Term
-sawscript> type m
+sawscript> return (type m)
```

Comment:
Personally, I would favor using `print (type m)` here instead of `return (type m)`, since I think it makes it more obvious what the intent of the command is.

(Similarly for other uses of `return` added elsewhere in this PR.)